src/sync/Synchroniser.js

```diff
@@ -57,7 +57,10 @@ export class Synchroniser {
     this.settings = settings;
     this.syncQueue = new SyncQueue(this.database);
     this.dispatch = dispatch;
-    this.extraHeaders = { 'msupply-site-uuid': DeviceInfo.getUniqueId() };
+    this.extraHeaders = {
+      'msupply-site-uuid': DeviceInfo.getUniqueId(),
+      'mobile-version': '5.0.0',
```

Comment:
should come from package.json, i.e. :https://github.com/openmsupply/mobile/blob/dfa74328694a98858568f0eb6e5e775eb9f81561/src/dataMigration.js#L33

Reply:
Yup, agreed.

864871feadfe5ee636f3a394ef52643853d26525.

Reply:
@wlthomson, shouldn't be merging an approved PRs
You also missed a console.log

Reply:
@andreievg Merged into release candidate due to time pressures. Main PR is still open: #2854.